airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -255,11 +255,16 @@ def execute(self, context: Context):
 
         return self.job_id
 
-    def execute_complete(self, context, event=None):
+    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> str:
+        if event is None:
+            err_msg = "Trigger error: event is None"
+            self.log.info(err_msg)
+            raise AirflowException(err_msg)
+
         if event["status"] != "success":
             raise AirflowException(f"Error while running job: {event}")
-        else:
-            self.log.info("Job completed.")
+
+        self.log.info("Job completed.")
```

Comment:
Similar to how you added more job specific information (i.e. the job id), you could do the same for all of these `Job completed` messages.

Reply:
I think it might be better if we do this in another PR. This PR intends to improve type annotation